src/systems/wheel_slip/WheelSlip.hh

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (C) 2019 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+#ifndef IGNITION_GAZEBO_SYSTEMS_WHEELSLIP_HH_
+#define IGNITION_GAZEBO_SYSTEMS_WHEELSLIP_HH_
+
+#include <ignition/gazebo/System.hh>
+#include <memory>
+
+namespace ignition
+{
+namespace gazebo
+{
+// Inline bracket to help doxygen filtering.
+inline namespace IGNITION_GAZEBO_VERSION_NAMESPACE {
+namespace systems
+{
+  // Forward declaration
+  class WheelSlipPrivate;
+
+  /// \brief This system applies a force to the first axis of a specified joint.
```

Comment:
Update documentation. This is a good place to explain the SDF parameters.

Reply:
a1b6d997612965bd16cdd64dabb09585855e1bdf

Added a todo for now until we figure out the specific units